.gitignore

```diff
@@ -11,4 +11,5 @@
 **/build
 **/src/*/generated
 **/src/*/generated_tests
-
+bundles/*.iml
```

Comment:
Didn't know if there was anywhere that we should be keeping iml files, so was being specific!